.teamcity/settings.kts

```diff
@@ -217,3 +208,70 @@ object ChocolateyGUIQA : BuildType({
         }
     }
 })
+
+object ChocolateyGUISign : BuildType({
+    id = AbsoluteId("ChocolateyGUISign")
+    name = "Chocolatey GUI (Script Signing)"
+
+    artifactRules = """
+    """.trimIndent()
+
+    params {
+        param("env.vcsroot.branch", "%vcsroot.branch%")
+        param("env.Git_Branch", "%teamcity.build.vcs.branch.Chocolatey_ChocolateyVcsRoot%")
```

Comment:
This isn't the right VCS Root.  Was this a copy/paste from the other kts file?  Will this be the same for all your PR's, where a new sign step was added?

```suggestion
        param("env.Git_Branch", "%teamcity.build.vcs.branch.ChocolateyGUI_ChocolateyGuiVcsRoot%")
```